ArchiveScraping/targetSpeaker.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+
+'''Strips description column for relevant information'''
+def findSpeaker():
+    try:
+        df = pd.read_csv('bloomberg.tv.2.csv')
+    except:
+        print("file could not be found in directory")
```

Comment:
Should this really be in a try/except block? If the file isn't found, the message will be printed to the console, then continue to the code below and probably fail there. The 'file doesn't exist' error message given by pandas should be fine.